flake.nix

```
@@ -1,6 +1,7 @@
 {
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.11";
+    nixpkgs-unstable.url = "github:NixOS/nixpkgs/nixos-unstable";
```

We can just follow the `master` nixpkgs branch to achieve this.

```suggestion
    nixpkgs-unstable.url = "github:NixOS/nixpkgs";
```